keps/993-two-phase-admission/README.md

```diff
@@ -253,6 +254,27 @@ The controller implementing a particular check should:
 * After approving the workload, keep an eye on the check if it starts failing,
   fail the check and cause the workload to move back to the suspended state.
 
+### Preemption Admission Check Controller
+
+Because now, the time when the eviction of the preemption candidate should vary based on the preemptor state
+the scheduler will not issue the eviction during it's process instead it will set a `Pending` admission check
+that is manged by a new built-in admission check controller.
+
+The **Admission Check Controller** will:
+
+- Watch for a change in state of the workloads pending preemption.
+- Watch for workloads that are finishing execution and therefore releasing quota.
+- Watch for AdmissionCheck changes, since the preemption policy can change.
+
+Since evaluating the current preemption state and the needs of a workload requires checking the sate of all resource pool the events are rate limited the kueue's cache is used.
+
+At every run the controller will get the list of workloads pending preemption.
+Since for some of these workloads is not necessary to issue eviction at that given point (eg. Having a pending check that uses AfterCheckPassedOrOnDemand policy) their quota reservation will be ignored.
+Fore every other preemption pending workloads, it will check if it can fit without evicting other workloads, case in which the preemption admission check condition will be set to `Ready`.
+If eviction of other workload is still needed, an updated list candidates is created and eviction is issued for all of them.
```

Comment:
Can you confirm if this is the implementation? and update the KEP accordingly.
In the first pass we would use the previously calculated set of preemption candidates. If this is not enough, we would recalculate the preemption candidates from scratch.

Nit: use a number list to show the steps of the algorithm.

Reply:
We don't keep track of previous lists, we are always "generating" an new list of candidates (some workload that is not present in a  previous eviction candidates list  might have ended its execution making the current list of candidates shorter).